python/paddle/fluid/data_feeder.py

```diff
@@ -116,3 +117,60 @@ def feed(self, iterable):
         for each_name, each_converter in six.zip(self.feed_names, converter):
             ret_dict[each_name] = each_converter.done()
         return ret_dict
+
+    def feed_parallel(self, iterable, num_places=None):
```

Comment:
`num_places` should not have a default value, because it is used but not checked whether it is `None` in this function.